reference/mscorlib.xml

```diff
@@ -64048,12 +64179,6 @@
                   <parameter name="monikerName" position="0" attrib="0" type="System.String" />
                 </parameters>
               </method>
-              <method name="BufferToBSTR(System.Array, System.Int32)" attrib="150" static="true" returntype="System.IntPtr">
```

Comment:
this was accidentally exposed, fixed with https://github.com/mono/mono/commit/813c1308fe53d16b419d22777f4d1124f06e7c9e